lighthouse-core/lib/page-functions.js

```diff
@@ -491,11 +492,11 @@ function getNodeDetailsImpl(element) {
 
   const details = {
     lhId,
-    devtoolsNodePath: getNodePath(element),
-    selector: getNodeSelector(htmlElement),
+    devtoolsNodePath: getNodePath(element) || '',
```

Comment:
I didn't catch the thread in chat about this, but I'm surprised we'd prefer empty strings here to null.

Reply:
null is better, except currently we feel pretty good that this is always `string` ... this is just attempting to enforce that. we don't really expect to get anything else here...

in other words, because this file is not typecheck, this is some fearsome-defensive-coding.

Reply:
maybe the fallbacks (if needed) should go in the individual functions rather than here?

Reply:
ehh let's just remove them.